### PR TITLE
Feature/react d3 graph

### PR DESCRIPTION
**Pull request by hrngoode-zz**
Please fill in this template.

- [ ] Type definition for react-d3-graph.

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,5 +31,6 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
-    }
+    },
+    "dependencies": {}
```
Comment (weswigham) [inline package.json]: Please revert these changes to the root `package.json`, thanks!

**Review by weswigham — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hrngoode) [reply package.json]: Apologies missed this!

**Review by weswigham — APPROVED** [review]
